src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,76 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany(fetch = FetchType.LAZY)
+    @JoinColumn(name = "ticket")
+    private List<Ticket> tickets;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "user_id")
+    private User user;
+    @Column(name = "date_time")
+    private LocalDateTime dateTime;
+
+    public Order() {
+    }
+
+    public Order(List<Ticket> tickets, User user, LocalDateTime dateTime) {
+        this.tickets = tickets;
+        this.user = user;
+        this.dateTime = dateTime;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public List<Ticket> getTickets() {
+        return tickets;
+    }
+
+    public void setTickets(List<Ticket> tickets) {
+        this.tickets = tickets;
+    }
+
+    public User getUser() {
+        return user;
+    }
+
+    public void setUser(User user) {
+        this.user = user;
+    }
+
+    public LocalDateTime getDateTime() {
+        return dateTime;
+    }
+
+    public void setDateTime(LocalDateTime dateTime) {
+        this.dateTime = dateTime;
+    }
+
+    @Override
+    public String toString() {
+        return "Order{"
+                + "tickets=" + tickets
+                + ", user=" + user
```

Comment:
Before, you specified that these fields (I mean tickets and user) have lazy type of fetching. Is it necessary to print them in toString() method?